src/unitTests/PinpointAnalytics.test.ts

```diff
@@ -61,20 +61,12 @@ describe("PinpointAnalytics", () => {
 		expect(error).toBeUndefined();
 	});
 
-	// Failure cases
-	/* disabled temporarily */
-	// it("should throw permission error on getEndpoint request", async () => {
-	// 	const error: any = await returnError(async () => {
-	// 		const command = new GetEndpointCommand({
-	// 			ApplicationId: PINPOINT_APPLICATION_ID,
-	// 			EndpointId: uuid.randomUUID()
-	// 		});
-	// 		await pinClient.send(command);
-	// 	});
-	// 	expect(error.$metadata.httpStatusCode).toBe(403);
-	// 	expect(error.message).toContain("not authorized to perform: mobiletargeting:GetEndpoint");
-	// });
+	it("should successfully get endpoint", async () => {
```

Comment:
This test seems to only call GetEndpoint and ensure no error return. It test that we have permission and can reach pinpoint application to call GetEndpoint but it does not really confirm that we sent proper endpoint data to pinpoint application. Can we test the later?

Reply:
Hey @vinguthh 
I can see in the [docs](https://docs.aws.amazon.com/pinpoint/latest/apireference/apps-application-id-events.html) that we have a POST method for Events but not a GET method.

In order to verify that correct data is sent we do that in our E2E by making sure that a response of status 203 is received when an event is posted against an endpoint, can you let me know if I'm missing something?